notebooks/18_deep_neural_network/index.md

```diff
@@ -0,0 +1,201 @@
+# Deep Neural Networks
+
+## Table of Content
+
+- [Introduction](#introduction)
+- [CNN Architecture](#CNN-Architecture)
```

Comment:
All mentioned layers and functions can be considered as subsections for this section

Reply:
This seems unresolved yet.
I mean this structure:
- CNN Architecture
      - Fully Connected Layers
      - Conv Layer
      ...